optimus/wdl_tasks/StarMkref.wdl

```diff
@@ -0,0 +1,28 @@
+
+task StarMkref {
```

Comment:
There is overlap here between this and https://github.com/HumanCellAtlas/skylab/blob/master/smartseq2_single_sample/build_references/build_star_reference/BuildStarReferenceBundle.wdl

There might be a good reason to keep them separate, but worth thinking about whether we can consolidate.

Reply:
I guess the big difference is that I needed to run a different version of star. We could pass the name of the docker image as a parameter and then I expect the two should be mergeable. 

it brings up a larger organizational question about how and where we put the tasks: perhaps we need a "general" location, as well as folders for 3' and full-length specific tasks?